### PR TITLE
add 2 png functions

### PR DESCRIPTION
**Pull request by ultimaRatio**
add basic read and write png functions (just had to give a filename).

### CODE REVIEW

**Comment by GuillaumeGomez** [comment]
Well, you just added FFI. It's useless as is.

**Comment by ultimaRatio** [comment]
ok, sorry but i only use ffi.

**Comment by ultimaRatio** [comment]
i've add the 2 small functions to the non-ffi code. 
You'll probably might want to change some thingss, as it may not be coherent with your code

File: src/image_surface.rs [hunk 1]
```diff
@@ -78,6 +78,24 @@ impl ImageSurface {
     pub fn get_width(&self) -> i32 {
         unsafe { ffi::cairo_image_surface_get_width(self.to_glib_none().0) }
     }
+    
+    #[cfg(feature = "png")]
+    pub fn to_png(&self, filename : String) -> Status {
```
Comment (GuillaumeGomez) [inline src/image_surface.rs]: `filename` should be a `&str` or a `&Path` or even:

```rust
pub fn to_png<S>(&self, filename: S) -> Status {
where S: AsRef<Path>
```

File: src/image_surface.rs [hunk 2]
```diff
@@ -78,6 +78,24 @@ impl ImageSurface {
...
+       use std::ffi::CString;
+       match CString::new(filename) {
+           Ok(file_name) =>  unsafe { ffi::cairo_surface_write_to_png(self.to_glib_none().0, file_name.as_ptr()) },
```
Comment (GuillaumeGomez) [inline src/image_surface.rs]: Since `file_name` isn't supposed to be a `String`, creating a `CString` is useless.

File: src/image_surface.rs [hunk 3]
```diff
@@ -78,6 +78,24 @@ impl ImageSurface {
...
+           _ => Status::WriteError
+       }
+    }
+    
+    #[cfg(feature = "png")]
+	pub fn from_png(filename : String) -> Option<ImageSurface> {
```
Comment (GuillaumeGomez) [inline src/image_surface.rs]: All same as above.

File: src/image_surface.rs [hunk 4]
```diff
@@ -78,6 +78,24 @@ impl ImageSurface {
...
+       use std::ffi::CString;
```
Comment (GuillaumeGomez) [inline src/image_surface.rs]: Indent broken.

**Comment by EPashkin** [comment]
Can't check it now, but normally gtk file functions has different names for windows.
Its reason why no file read\write function in gtk (if I not wrong)

**Pull request closed without merging**